draft-zip-hybrid-deferred-devfund.md

```diff
@@ -0,0 +1,413 @@
+accommodate the disbursement mechanism defined in a future ZIP. This ensures
+that the system can evolve based on feedback and new developments in the Zcash
+ecosystem.
+
+**Non-Direct Funding Model**: This ZIP recognizes there is strong community
+support for a non-direct funding model.  As such, ECC and ZF MUST collaborate
+with the Zcash community to research and explore the establishment of a
+non-direct funding model. The research should consider potential designs as
+well as possible legal and regulatory risks.
+
+**ZCG Slice**: The ZCG slice should encourage decentralization and support
+independent teams dedicated to Zcash. Major funding decisions should be based,
+to the extent feasible, on inputs from domain experts and pertinent
+stakeholders.
+
+**Value of ZEC**: In case the value of ZEC jumps, ZCG should not wastefully use
+excessive amounts of funds. Conversely, given market volatility and eventual
+halvings, it is desirable to create rainy-day reserves.
+
+**Funding Stream**: The ZCG funding stream should conform to the standards of
+ZIP 207 and ZIP 214.
+
+## Specification
+
+Consensus changes implied by this specification are applicable to the Zcash
+Mainnet. Similar (but not necessarily identical) consensus changes SHOULD be
+applied to the Zcash Testnet for testing purposes.
+
+### Hybrid Deferred Dev Fund Allocation
+
+The Zcash Development Fund SHALL be enacted in Network Upgrade 6 and last for 2
+years. The Dev Fund would consist of 20% of the block subsidies, split into 2
+slices:
+
+* 60% to a deferred Dev Fund “lockbox,” which will be allocated in the future
+  once consensus is reached on fund allocation or a new non-direct funding
+  model is established and implemented.
+* 40% to Zcash Community Grants to fund independent teams in the Zcash ecosystem.
+
+### Deferred Dev Fund Lockbox
+
+<u>Deferred Development Fund Chain Value Pool Balance</u>
+
+A new development fund value pool balance, denominated in zatoshis, will be
+created. This balance will be initialized to zero at the NU6 activation block.
+
+<u>Coinbase Transaction Adjustments</u>
+
+For coinbase transactions, a specified proportion of the block subsidy will be
+added to the deferred development fund chain value pool balance and subtracted
+from the transparent transaction value pool. This subtraction will occur after
+block reward value is logically added to the transparent value pool. The exact
+proportions (numerator/denominator) of the block subsidy allocated to the
+deferred development fund must be defined to ensure the correct allocation at
+every block.
+
+<u>Proportion Specification</u>
+
+The proportions of the block subsidy allocated to the deferred development fund
+and any existing funding streams must add up to the intended total at every
+block. Before NU6 activation, the actual percentage allocated to the deferred
+development fund will be zero. After NU6 activation, |percentage| of the block
```

Comment:
The abstract, and the beginning of the specification, have defined percentages; here this appears to instead imply a parameter.

For the purposes of today's deadline:
- If this is intended to be a parameter, define the parameter at the start of the spec section (before its first use), and use it consistently everywhere.
- If it is not intended to be a parameter, replace this with 12% to be consistent with the rest of the ZIP draft.

Reply:
It should be 12%, consistent with the rest of the ZIP draft.